src/ngraph/runtime/cpu/pass/cpu_rnn_fusion.cpp

```diff
@@ -62,6 +64,139 @@
     }
 
 using namespace ngraph;
+
+void ngraph::runtime::cpu::pass::LSTMFusion::construct_onnx_lstmcell_fprop()
+{
+    size_t ref_batch_size = 2;
+    size_t ref_input_size = 3;
+    size_t ref_hidden_size = 3;
+    size_t ref_gates_count = 4;
+
+    auto X =
+        std::make_shared<pattern::op::Label>(element::f32, Shape{ref_batch_size, ref_input_size});
+    auto W = std::make_shared<pattern::op::Label>(
+        element::f32, Shape{ref_gates_count * ref_hidden_size, ref_input_size});
+    auto R = std::make_shared<pattern::op::Label>(
+        element::f32, Shape{ref_gates_count * ref_hidden_size, ref_hidden_size});
+    auto H_t =
+        std::make_shared<pattern::op::Label>(element::f32, Shape{ref_batch_size, ref_hidden_size});
+    auto C_t =
+        std::make_shared<pattern::op::Label>(element::f32, Shape{ref_batch_size, ref_hidden_size});
+
+    auto ref_lstm_cell = std::make_shared<op::LSTMCell>(X, W, R, H_t, C_t, ref_hidden_size);
+
+    auto callback = [X, W, R, H_t, C_t](pattern::Matcher& m) {
+
+        auto pattern_map = m.get_pattern_map();
+        ngraph::runtime::cpu::rnn_utils::rnntype rnn_type =
+            ngraph::runtime::cpu::rnn_utils::rnntype::vanilla_lstm;
+
+        auto target_lstm_node = m.get_match_root();
+        auto lstmcell_op = std::dynamic_pointer_cast<op::LSTMCell>(m.get_match_root());
+        auto src_iter =
+            std::make_shared<ngraph::op::Concat>(NodeVector{pattern_map[H_t], pattern_map[C_t]}, 0);
+        auto bias_iofc = target_lstm_node->get_argument(5);
```

Comment:
do we need to check if Bias is present? 
i don't see Bias in line 86.

Reply:
@gaurides , LSTMCell ctor adds default `bias` and `peephole` input which are constant  initialized if they are not given implicitly.